tests/e2e/cucumber/steps/cli.ts

```diff
@@ -0,0 +1,25 @@
+import { Given, DataTable } from '@cucumber/cucumber'
+import { World } from '../environment'
+import { execSync } from 'child_process'
+
+Given(
+  '{string} creates following files(s) using CLI:',
+  function (this: World, stepUser: string, stepTable: DataTable) {
+    const id = this.usersEnvironment.getCreatedUser({ key: stepUser }).uuid
+
+    const containerIdOrName = 'opencloud'
+    const storagePath = `/var/lib/opencloud/storage/users/users/${id}`
```

Comment:
can we make these values variable?

Reply:
Probably you mean `const containerIdOrName = 'opencloud'` ?
if yes, I'll do it. first I want to adjust this test in the CI